x-pack/plugins/lens/public/trigger_actions/open_lens_config/helpers.scss

```diff
@@ -1,6 +1,9 @@
 // styles needed to display extra drop targets that are outside of the config panel main area while also allowing to scroll vertically
 .lnsConfigPanel__overlay {
   clip-path: polygon(-100% 0, 100% 0, 100% 100%, -100% 100%);
```

Comment:
@dej611 

> I think the original tab was only part of the issue, as the current remediation works for the initial flyout but as soon as one dimension is created/edited the new tab is taking up all the screen area:
> ![responsive_flyout_issue](https://github.com/elastic/kibana/assets/924948/739f010b-c344-4be4-b6ba-0f7038f49738)


My response:
I thought it was expected? What do we want to do instead?

Reply:
I managed to fix it to occupy only part of the space, hope it's fine!

<img width="923" alt="Screenshot 2024-05-03 at 10 57 40" src="https://github.com/elastic/kibana/assets/4283304/5c6650a2-e183-4999-ae5a-26051b694b75">

Reply:
this is solved in https://github.com/elastic/kibana/pull/182376/commits/2b38b0f187a522382c70090aa00e174f1c663b12